package.json

```diff
@@ -108,7 +108,7 @@
     "prismock": "^1.21.1",
     "resize-observer-polyfill": "^1.5.1",
     "tsc-absolute": "^1.0.0",
-    "typescript": "^4.9.4",
+    "typescript": "^5.0.0",
```

Comment:
better to be explicit :)
```suggestion
    "typescript": "^5.6.3",
```

Reply:
or even 
```suggestion
    "typescript": "5.6.3",
```

as ts doesn't follow semver

Reply:
Applied this recommendation